src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -286,10 +288,10 @@
     </script>
 
     <script type="text/template" id="template-sidebar-plugin">
-            <a class="plugin-launcher i18n" data-i18n="[title]<%=fullName%>" href="javascript:;">
-                <img src="<%- pluginSrcFolder %>/icon<%= selected ? '_active' : ''%>.png">
-                <div class="i18n" data-i18n="<%- pluginObject.toolbarName %>"><%- pluginObject.toolbarName %></div>
-            </a>
+        <button class="nav-apps-button" i18n" data-i18n="[title]<%=fullName%>">
```

Comment:
Good catch. I just ran across this while working on #686. I'm going to fix it there as it has implications in the plugin code.